static/js/ui_init.js

```diff
@@ -263,7 +263,10 @@ export function initialize_kitchen_sink_stuff() {
         // preventDefault, allowing the modal to scroll normally.
     });
 
-    $(window).on("resize", _.throttle(resize.handler, 50));
+    $(window).on(
+        "resize",
+        _.throttle(() => setTimeout(resize.handler, 0), 50),
+    );
```

Comment:
I was not able to reproduce the bug, but I understand the possibility, maybe this change can help?